tests/test_autora_theorist_avengers.py

```diff
@@ -12,7 +12,7 @@ def test_fit():
     y = np.array([1, 2, 3])
     model.fit(X, y)
     assert model.coefficients is not None
-    assert len(model.coefficients) == 10  # 1 intercept + 2 linear + 2 log + 5 quadratic terms
+    assert len(model.coefficients) == 8  # 1 intercept + 2 linear + 2 log + 4 quadratic terms
```

Comment:
Shouldn't the len(model.coefficients) be == 9 because there is 1 intercept, 2 linear terms, 2 log terms and 4 quadratic terms? So, (1+2+2+4 = 9)